src/ui/assets/AssetsListContainer.js

```diff
@@ -47,37 +53,58 @@ export const mapStateToProps = state => ({
 });
 
 export const mapDispatchToProps = dispatch => ({
-  onDidMount: (params) => {
-    dispatch(fetchAssets(params));
+  onDidMount: () => {
+    dispatch(setListFilterParams({}));
+    dispatch(fetchGroups({ page: 1, pageSize: 0 }))
+      .then(() => dispatch(fetchAssetsPaged()));
```

Comment:
this was because of my update that asset item needing the group info in selectors, but I already made changes and add fallback to the group object so this solves the problem